pyrasterframes/src/main/python/pyrasterframes/__init__.py

```diff
@@ -110,16 +110,33 @@ def _aliased_writer(df_writer, format_key, path, **options):
 
 def _raster_reader(
         df_reader,
-        path=None,
-        catalog=None,
+        source=None,
         catalog_col_names=None,
         band_indexes=None,
         tile_dimensions=(256, 256),
         lazy_tiles=True,
         **options):
+    """
+    Returns a Spark DataFrame from raster data files specified by URI pointers
```

Comment:
End with `.`?

Also, "pointers" may be loaded... can it just say "by URIs"?

Reply:
Note: I don't really know the convention with sentences/periods in Python docs. My inclination would be to provide standard grammatical termination.